src/domains/citizen-auth-app/env/weu-prod01/terraform.tfvars

```diff
@@ -51,9 +51,9 @@ cidr_subnet_fnlollipop              = ["10.0.17.0/26"]
 function_lollipop_kind              = "Linux"
 function_lollipop_sku_tier          = "PremiumV3"
 function_lollipop_sku_size          = "P1v3"
-function_lollipop_autoscale_minimum = 1
-function_lollipop_autoscale_maximum = 3
-function_lollipop_autoscale_default = 1
```

Comment:
It's better to align them to fast-login ones

Reply:
aligned in https://github.com/pagopa/io-infra/pull/825/commits/db70e808b02e03503aa819f31be84984a5b5ecfb